### PR TITLE
[Content] Coalition: Big Finisher Launcher

### PR DESCRIPTION
**Pull request by Arachi-Lover**
----------------------
**Content (Artwork /Outfit)**

## Summary
With the addition of the "Pod" outfits, smaller versions of the various missile launchers in human space, which allow smaller ships to make use of weapons otherwise out of their reach (ie Typhoons), some discussion on the Discord led to me thinking how the same can't really be done for Finisher Torpedoes, for their launcher already is the Finisher **Pod**. The obvious solution thus is to make it clear that that *is* the smaller version, and this PR adds the large version of the Finisher Torpedo Launcher.

I looked at Saryd ship names to try and make up a word that sounded Saryd enough, and used that for the name of the outfit (Finisher Maegrolain), since I didn't think that any of the other naming schemes we use for the large launchers would really be appropriate. Also to add some more variety.

The launcher itself is one that fires in bursts, having a `burst count` of 20, and 120 capacity for Finisher Torpedoes, allowing for 6 bursts by default. The burst damage is pretty high, with a full burst taking out about 90% of a Punisher's shields (no AM interference, all shots hit).
It's also very big, and only 3 Heliarch ships have variants fitted with it: Punisher, Interdictor and Hunter.

In order to make it so that the Punisher could fit 4 of these (because otherwise it'd either have only 2, with a lot of space going unused, or worse yet, have 3 and not be symmetrical), its weapon capacity was brought up from 324 to 352.

## Artwork Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I created a PR to the [endless-sky-assets repo](https://github.com/EndlessSkyCommunity/endless-sky-assets) with the necessary image, blend, and texture assets: https://github.com/EndlessSkyCommunity/endless-sky-assets/pull/49
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/261
 
 The outfit image was done by Geffulte. Thanks again for that!

### CODE REVIEW

File: data/coalition/coalition weapons.txt [hunk 1]
```diff
@@ -77,6 +77,45 @@ effect "bombardment impact"
 	"velocity scale" -.2
 
 
+outfit "Finisher Maegrolain"
+	category "Secondary Weapons"
+	licenses
+		Heliarch
+	cost 7260000
+	thumbnail "outfit/finisher maegrolain"
+	"mass" 40
```
Comment (warp-core) [inline data/coalition/coalition weapons.txt:86]: Outfit space used = 88
Finisher capacity = 120
Finisher mass = 0.5
120 * 0.5 = 60
88 - 60 = 28
```suggestion
	"mass" 28
```

**Review by warp-core — COMMENTED** [review]
Regarding the weapon capacity change on the Punisher, could you not just give it 2 of these and some other weapons?

File: data/coalition/coalition weapons.txt (hunk 1 above)
Comment (Arachi-Lover) [reply data/coalition/coalition weapons.txt:86]: Didn't know there was a calc for it. Thanks.

**Comment by Arachi-Lover** [comment]
> Regarding the weapon capacity change on the Punisher, could you not just give it 2 of these and some other weapons?

Yes, but I'd rather have the variants for this be specialized, rather than mix and matching with more of the Heliarch arsenal.

File: data/coalition/coalition weapons.txt [hunk 2]
```diff
@@ -77,6 +77,45 @@ effect "bombardment impact"
...
+		"firing energy" 184
+		"firing heat" 76
+		"acceleration" .8
+		"drag" .1
+		"turn" 0.1
+		"homing" 2
+		"radar tracking" .8
+		"missile strength" 50
+		"submunition" "Active Finisher" 1
+	description `Baptized with the Saryd word for "furious thunderstorm," this enormous launcher is not meant for prolonged fights nor for use by lighter craft, but for delivering a few quick, devastating bursts of Finisher torpedoes, having much greater damage potential than its smaller counterpart.`
```
Comment (roadrunner56) [inline data/coalition/coalition weapons.txt:117]:
```suggestion
	description `Baptized with the Saryd word for "furious thunderstorm," this enormous launcher is not meant for prolonged fights nor for use by lighter craft, but for delivering a few quick, devastating bursts of Finisher torpedoes. It has much greater damage potential than its smaller counterpart.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by roadrunner56 — APPROVED** [review]
Description looks good.

**Comment by Amazinite** [comment]
The weapon capacity change seems fine to me. It puts the Punisher from being just above the Cruiser in weapon capacity to just below the Dreadnought.

File: data/coalition/coalition weapons.txt [hunk 3]
```diff
@@ -77,6 +77,45 @@ effect "bombardment impact"
 	"velocity scale" -.2
 
 
+outfit "Finisher Maegrolain"
```
[the remark this answers is not in the document]
Comment (Amazinite) [reply data/coalition/coalition weapons.txt:80]: Actually I'll just move it myself because this otherwise looks good to me.

**Pull request merged**